Training Modules/Introduction to Numpy/Introduction to Numpy.md

```diff
@@ -0,0 +1,637 @@
+![Numpy](https://miro.medium.com/max/765/1*cyXCE-JcBelTyrK-58w6_Q.png)
+# NumPy
+NumPy **(Numerical Python)** is an open source Python library that’s used  for working with numerical data in Python.
+It is a general-purpose array-processing package. It provides a high-performance multidimensional array object, and tools for working with these arrays. It is the fundamental package for scientific computing with Python.
+
+Numpy can also be used as an efficient multi-dimensional container of generic data.
+
+## Pre-Requisites
+* Basic working knowledge of Python.
+
+## Basic things to know before we start with the topic:
+In the Numpy definition, you read that they basically work with **Arrays** so what are **Arrays**???
+### Arrays
```

Comment:
Have you taken this from some book, website, blog ... be sure to mention the credits in that case.